docs/docs/guide/debugging.mdx

```diff
@@ -0,0 +1,303 @@
+```
+
+Another side effect is that Reanimated uses web implementations of all
+functions. This means that the `scrollTo` function will work
+(using the native web implementation), but the `measure`
+function will not be available, and its usage will trigger this error:
+
+```
+[reanimated.measure] method cannot be used for web or Chrome Debugger
+```
+
+You may still use the standard web version of measure as described
+[here](../api/nativeMethods/measure).
+
+Those functions that are provided by Reanimated and do not have web
+implementations won't work at all.
+An example of this behavior is the `useAnimatedSensor` hook,
+which currently only works on mobile platforms. When debugging in Chrome and using
+this hook, the following message will appear in the logs:
+
+```
+[Reanimated] useAnimatedSensor is not available on web yet.
+```
+
+But despite all of this, it is still possible to set breakpoints both in
+normal JS code and in worklets (since they run on the main JS
+thread now).
+
+### Chrome DevTools
+
+<ChromeDevToolsTable />
+
+**Summary:**
+Works and both contexts can be debugged.
+
+:::warning
+
+This is an early experimental feature.
+
+:::
+
+Worklet debugging is now available in Reanimated apps as an early experimental
+feature. We do not recommend using Chrome DevTools to debug the UI context, but
+instead try using the Flipper tool. It has fewer bugs and provides a much better
+experience.
+
+If you still want to use Chrome DevTools then follow these steps:
+
+1. Add the `patch-package` to your project and set it up to run as a `post-install` task.
+More details can be found [here](https://www.npmjs.com/package/patch-package).
+
+2. Add [this](/content/metro-inspector-proxy+0.72.1.patch) patch for `metro` and
+[this](/content/react-native-gesture-handler+2.5.0.patch) one for `react-native-gesture-handler`
+to the `patches/` folder. *Warnings about mismatched patch versions can
+be safely ignored if the patch was successfully applied*.
+
+3. Go to [chrome://inspect](chrome://inspect) in a Google Chrome browser.
+
+4. Select the Reanimated runtime from the list. The list should look something
+like this:
+
+![Screenshot showing Chrome DevTools target selection](/img/debugging/ChromeDevToolsList.png)
+
+You may choose either `Reanimated Runtime` or `Reaniamted Runtime experimental
+(Improved Chrome Reloads)`, but we recommend the latter.
+
+_Debugging relies on source maps that are generated by the babel plugin, so you
+might have to run `yarn start --reset-cache` for those changes to take effect.
+In case it still doesn't work after that please reinstall the app and reset metro
+cache once again._
+
+:::caution
+
+Known issues include:
+
+- Reloads don't work - if a debugger is connected to the Reanimated Runtime while
+the reload is performed, the app will crash
+
+- On iOS, a breakpoint can't be set in a line where a breakpoint was previously
+set and then removed
+
+- On iOS, breakpoint labels are not visible
```

Comment:
Is this still relevant?

Reply:
This was fixed in Flipper lately but the issue is still present in Chrome DevTools